LawGuardPro.Application/DTO/UserUpdateDTO.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LawGuardPro.Application.DTO;
```

Comment:
Always put a space between class declaration and namespace

Reply:
Done.